lxc/copy.go

```diff
@@ -304,6 +321,47 @@ func (c *cmdCopy) copyContainer(conf *config.Config, sourceResource string,
 			dest = dest.UseTarget(c.flagTarget)
 		}
 
+		if c.flagRefresh {
+			// Make sure the target container exists if using --refresh
```

Comment:
Why not just copy the full container if there is no existing container on target?